src/main/java/lotto/controller/PurchaseController.java

```diff
@@ -0,0 +1,27 @@
+package lotto.controller;
+
+import lotto.domain.lottery.Buyer;
+import lotto.domain.lottery.Lottos;
+import lotto.dto.LottoNumberResponses;
+import lotto.view.output.PurchaseWriter;
+
+public class PurchaseController {
+    private PurchaseController() {
+
+    }
+
+    public static Lottos purchase(final Buyer buyer) {
+        final int ticketCount = buyer.getTicketCount();
+
+        Lottos lottos = Lottos.from(ticketCount);
+
+        LottoNumberResponses lottoResponses =
+                LottoNumberResponses.build(buyer, lottos);
+
+        PurchaseWriter.printNewLine();
+        PurchaseWriter.printPurchaseCount(lottoResponses);
```

Comment:
lottoResponses가 dto라 상관은 없겠지만 이때 파라미터로 lottoResponses.purchaseCount()를 바로 전달해도 좋을 것 같아요

Reply:
계층 간 값을 전달할 때, 값 타입 원소가 아니라 객체로 통신해라!
라는 이야기를 어디선가 풍문으로 전해들은 기억이 있습니다.

객체로 포장되어 있는 내용의 캡슐화를 깨고,
불필요한 Getter 사용을 조장하기 때문이라고 생각해요.

이 부분에 대해서는 어떻게 생각하시는지 궁금해요!
@jisu-om

Reply:
아 그렇게 볼 수도 있겠네요! 
저는 뭔가 뷰에 가서 한번 더 이 객체의 메서드를 호출해야 해서 그것보다는 필요한 자료만 딱 던져주는게 좋을꺼라고 생각했어요. 
사실 저도 객체를 전달하였는데 이것이 좀 그렇다는 코드리뷰를 받았습니다 ㅎㅎ 


... 지피티한테 빠르게 물어봤는데 객체를 파라미터로 보내면 객체의 상태나 행동을 노출시키는 거라 캡슐화를 저해한다고 하네요 
(참고겸 그대로 올려드립니당!)
Q
PurchaseWriter.printPurchaseCount(lottoResponses);
PurchaseWriter.printPurchaseCount(lottoResponses.purchaseCount());
Which one is a better fit for object oriented programming?

A
From an object-oriented programming perspective, the second option, PurchaseWriter.printPurchaseCount(lottoResponses.purchaseCount());, is the better fit.
In object-oriented programming (OOP), one of the main principles is encapsulation, which is about hiding the internal state and behavior of an object and exposing only what is necessary. In this case, lottoResponses.purchaseCount() is an example of a method that encapsulates the behavior of the LottoResponses object. This method presumably calculates and returns the purchase count, which is then passed to the printPurchaseCount() method of the PurchaseWriter class.
On the other hand, the first option, PurchaseWriter.printPurchaseCount(lottoResponses);, passes the entire LottoResponses object to the printPurchaseCount() method. This could potentially violate the principle of encapsulation if the printPurchaseCount() method directly accesses or modifies the state of the LottoResponses object.
Therefore, the second option is a better fit for object-oriented programming because it adheres more closely to the principle of encapsulation. However, the actual suitability may also depend on the specific behavior of these methods and the design of your program.